retworkx/__init__.py

```diff
@@ -626,6 +626,40 @@ def _graph_all_pairs_dijkstra_shortest_path(graph, edge_cost_fn):
     return graph_all_pairs_dijkstra_shortest_paths(graph, edge_cost_fn)
 
 
+@functools.singledispatch
+def all_pairs_all_simple_paths(graph, min_depth=None, cutoff=None):
+    """Return all the simple paths between all pairs of nodes in the graph
+
+    This function is multithreaded and will launch a thread pool with threads
+    equal to the number of CPUs by default. You can tune the number of threads
+    with the ``RAYON_NUM_THREADS`` environment variable. For example, setting
+    ``RAYON_NUM_THREADS=4`` would limit the thread pool to 4 threads.
+
+    :param PyGraph graph: The graph to find all simple paths in
```

Comment:
```suggestion
    :param graph: The graph to find all simple paths in. This can be a :class:`~retworkx.PyGraph`
        or a :class:`~retworkx.PyDiGraph`
```